lib/get_ip.py

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env python
```

Comment:
I would like to avoid storing a Python script in the lib folder. Bash-it is a Bash framework, and we should try to minimize dependencies as much as possible.

Take a look at the `ips` function at https://github.com/Bash-it/bash-it/blob/master/plugins%2Favailable%2Fbase.plugin.bash#L4 for an example on how to get all of the system's IP addresses. This could probably be adapted into a second function that only returns the IP address that you're looking for.

There's a long thread of possible solutions for this using Bash-only tools here: http://stackoverflow.com/questions/13322485/how-to-i-get-the-primary-ip-address-of-the-local-machine-on-linux-and-os-x

Some things to keep in mind:

* Needs to work on a wide array of operating systems.
* Not all of them have the same tools (`ip`/`ifconfig`) or the same version (e.g. `hostname -i` is not available on OS X)
* Assume that things like Python are optional and not available on some systems by default